guac_actix/Cargo.toml

```diff
@@ -4,11 +4,17 @@ version = "0.1.0"
 authors = ["Ben Wang <wangben3@gmail.com>"]
 
 [dependencies]
-actix = "0.5.7"
-actix-web = "0.6.7"
+actix = "0.7"
+actix-web = {version = "0.7",  default-features = false}
```

Comment:
Are we just avoiding ring here?

Reply:
we need ring anyways for parity stuff, but we don't depend on brotli and lots of the other stuff actix-web brings in and we don't want